plugins/rikki/loungestatistics/components/SlothStatistics.php

```diff
@@ -183,13 +213,15 @@ public function calculateStats($season)
 
     public function onSeasonChange()
     {
-        if (input('season_id') == -1) {
-            $this->calculateStats(null);
+        $season_id = input('season_id');
+        if ($season_id == -1) {
+            $this->selectedSeason = null;
         } else {
-            $this->calculateStats(Season::find(input('season_id')));
+            $this->selectedSeason = Season::find($season_id);
         }
+        $this->calculateStats($this->selectedSeason);
         return [
-            '#slothstatistics' => $this->renderPartial('@stats')
+            '#teamstatistics' => $this->renderPartial('@stats')
```

Comment:
Should this not stay as sloth, since these aren't team statistics?

Reply:
Whoops, copy paste error, thanks!